### DISCUSSION TITLE
Persistent dates when switching between sites

### DISCUSSION THREAD

**Discussion by ghost** [plausible/analytics#1884] [Anything Else]
I prepare weekly analytics reports for multiple different sites. It would be great if the calendar/date picker would be persistent across sites.

For example:

1. Select site A.
2. Select a custom date range.
3. Change to site B.
4. Currently, the date range resets to 'Last 30 days' – it would be convenient if the date range set in step 2 could persist.

Thanks team!

**Comment by metmarkosaric** [comment]
thanks for sharing @ario5168! interesting idea! currently, we save the previously selected option so it's the default one you see on your next visit too. so if you use the same date range every time, the way it works now should be fine for you already

**Comment by ghost** [comment]
Thanks @metmarkosaric – unfortunately it doesn't seem to save to me: whichever date range I choose (custom or one of the preset options), it defaults back to 'Last 30 days' when I switch to another site or log out and log in again. I would love for it to save the previously selected option :)

**Reply by metmarkosaric** [reply]
hmm that's weird. are you clearing your cookies?

**Reply by ghost** [reply]
I'm not clearing any data. My default browser, Safari, is set to retain history for one week. I've disabled content blockers for plausible.io. I've also tried with a fresh install of Chrome (using all default settings, no extensions) and still experience the same issue unfortunately.

**Reply by metmarkosaric** [reply]
hmm ok. thanks for sharing! i normally use firefox and now i tested on chrome too and in both cases it saves my last choice fine when i select any of the default time ranges. we'll need to take a closer look at this to figure out what's going on on other devices/browsers.
